packages/fuel-indexer-types/src/lib.rs

```diff
@@ -29,11 +29,46 @@ pub type UInt16 = u128;
 pub type Timestamp = u64;
 pub type Charfield = String;
 pub type Boolean = bool;
-pub type Blob = Vec<u8>;
+
+#[derive(Deserialize, Serialize, Clone, Eq, PartialEq, Debug, Hash)]
+pub struct Blob(pub Vec<u8>);
+
+impl From<Vec<u8>> for Blob {
```

Comment:
👌🏽 

Can we also add `TryFrom` here as well?

Reply:
I had initially just used `TryFrom` but then just went with `From` as I couldn't think of what kind of validation we would want here, and what could produce the error, as it's just 
```
impl From<Vec<u8>> for Blob {
    fn from(value: Vec<u8>) -> Self {
        Blob(value)
    }
}
```